### PR TITLE
chore: use docker-compose to start etcd and layotto #635

### PR DESCRIPTION
**Pull request by YoungMa962**
* add docker compose

* modify the etcd quickstart doc

**What this PR does**:
add use docker-compose to start etcd and layotto 

**Which issue(s) this PR fixes**:
issue #635

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/zh/start/configuration/start.md [hunk 1]
```diff
@@ -4,8 +4,23 @@
...
 
-## 启动 etcd
+```bash
+cd docker/layotto-etcd
+# Start etcd and layotto with docker-compose
+docker-compose up -d
+```
+
+#### **本地编译（不适合 Windows)**
+您可以使用 Docker 运行 MinIO，然后本地编译、运行 Layotto。
```
Comment (seeflood) [inline docs/zh/start/configuration/start.md:19]:
```suggestion
您可以使用 Docker 运行 etcd，然后本地编译、运行 Layotto。
```

File: docs/zh/start/configuration/start.md [hunk 2]
```diff
@@ -4,8 +4,23 @@
 本示例架构如下图，启动的进程有：客户端程程序、Layotto、etcd 。
 
 ![](https://gw.alipayobjects.com/mdn/rms_5891a1/afts/img/A*dzGaSb78UCoAAAAAAAAAAAAAARQnAQ)
+### step 1. 启动 etcd 和 Layotto
```
Comment (seeflood) [inline docs/zh/start/configuration/start.md:7]: Hi, could u also modify the sequencer quickstart docs?
They use etcd too:
https://mosn.io/layotto/#/zh/start/sequencer/start
https://mosn.io/layotto/#/en/start/sequencer/start

**Review by seeflood — CHANGES_REQUESTED** [review]

File: docs/zh/start/configuration/start.md [hunk 3]
```diff
@@ -4,8 +4,23 @@
...
+<!-- tabs:start -->
+#### **使用 Docker Compose**
```
Comment (seeflood) [inline docs/zh/start/configuration/start.md:9]: Could u also modify the english version of this doc?
https://mosn.io/layotto/#/en/start/configuration/start

**Review by seeflood — CHANGES_REQUESTED** [review]

**Comment by seeflood** [comment]
Thanks for your contribution!

File: docs/zh/start/configuration/start.md (hunk 3 above)
Comment (YoungMa962) [reply docs/zh/start/configuration/start.md:9]: no problem

**Review by seeflood — APPROVED** [review]
lgtm

**Review by zhenjunMa — APPROVED** [review]

**Pull request merged**